UPGRADE.md

```diff
@@ -1,4 +1,24 @@
 # UPGRADING
+## Recommended way of upgrading
+* upgrade locally first. After you fix all issues caused by the upgrade, version them and then continue with upgrading application on a server
+* upgrade one version at a time:
+    * Start with a working application
+    * Upgrade to the next version
+    * Fix all issues
+    * Repeat
+* typical upgrade sequence should be:
+    * `docker-compose down`
+    * follow upgrade notes for `docker-compose.yml`, `Dockerfile`, docker containers
+    * `docker-compose up -d`
+    * update shopsys framework dependencis in `composer.json` to version you are upgrading to
+    * `composer update`
+    * follow all upgrade notes you didn't do yet
```

Comment:
I believe better formulation is "follow all upgrade notes you **have not done yet**"